sqlmesh/core/context.py

```diff
@@ -1974,6 +1979,7 @@ def _run(
         select_models: t.Optional[t.Collection[str]],
         circuit_breaker: t.Optional[t.Callable[[], bool]],
         no_auto_upstream: bool,
+        is_run_command: bool = False,
```

Comment:
`_run` is only ever called by the `run` command. We don't need to pass this argument here.